run_1_tdx_gpkg_to_geoparquet.py

```
@@ -49,11 +51,9 @@ def _calculate_geodesic_length(line) -> float:
         out_file_name = os.path.join(gpq_dir, os.path.basename(gpkg).replace('.gpkg', '.parquet'))
         if os.path.exists(out_file_name):
             continue
-
-        gdf = gpd.read_file(gpkg)
-        gdf['LINKNO'] = gdf['LINKNO'].astype(int) + (tdx_header_number * 10_000_000)
-
+        
         if 'streamnet' in os.path.basename(gpkg):
+            gdf['LINKNO'] = gdf['LINKNO'].astype(int) + (tdx_header_number * 10_000_000)
```

Ruff (0.8.0)

Undefined name `gdf`

(F821)